.travis.yml

```diff
@@ -72,6 +72,7 @@ services:
   - redis
 
 before_install:
+  - '$(php PruneTest.php'
```

Comment:
Looks weird. What `$(` means?

Reply:
Where's the closing `)`?

Reply:
Hmm there's a missing brace :|
The script will for example this:
```
export FRAMEWORK=skip
```

The `$(...)` makes bash interpret that and actually changes the environment.